oak_containers_orchestrator/src/container_runtime.rs

```diff
@@ -79,7 +80,10 @@ async fn rmount_dir(source: PathBuf, target: PathBuf) -> Result<(), anyhow::Erro
     Ok(())
 }
 
-pub async fn run(container_bundle: &[u8]) -> Result<(), anyhow::Error> {
+pub async fn run(
+    container_bundle: &[u8],
+    shutdown_sender: Sender<()>,
```

Comment:
nit: would it make sense to rename to `shutdown_orchestrator_sender` (and below)? I was not sure who gets shutdown here at first.

Or, alteratively, a more general: `control_orchestrator_sender` with a Shutdown signal?

Reply:
Renamed it to "exit_notification_sender", since the container runtimes notifies the gRPC service the container has exited.